src/lando/main/scm/abstract_scm.py

```diff
@@ -27,6 +27,11 @@ def scm_name(cls) -> str:
         """Return a _human-friendly_ string identifying the supported SCM (e.g.,
         `Mercurial`)."""
 
+    @classmethod
+    def get_rejects_path(cls) -> Path:
+        """Return the path where the SCM stores reject files."""
+        return Path(".")
```

Comment:
nit: can you add an explanation why this is set as the default? Is it ever used?

Reply:
Will do. It's used by the landing worker to locate the REJECTS, as Git and Hg put them in different locations https://github.com/mozilla-conduit/lando/pull/182/files#diff-15a775db92cbe9bf7e735238755c515149c96ba8bebb70ca8d0bbb1e0f9657daR164...

But the local directory seems like a reasonable default.